### ISSUE TITLE
Support initialization of an arbitrary-backend Tensor from a numpy array or nested list

### ISSUE THREAD

**Issue by alewis** [google/TensorNetwork#843]

**Comment by Thenerdstation** [comment]
Numpy array should always work given you use `backend.convert_to_tensor`. Nested lists behave differently for each backend so that might be harder.

**Comment by alewis** [comment]
Fair enough. Probably just from an array is fine.

**Comment by MikeMerz** [comment]
Hey,
Is this issue still still open?

**Comment by alewis** [comment]
Yep, all yours. I think we aren't going to support nested lists, but it would be good if initialization in an arbitrary backend could be done from a numpy array:

`
A = np.zeros(shape, dtype)
At = tn.Tensor(A, backend="numpy") # produces a tn.Tensor with the numpy backend
At_tf = tn.Tensor(A, backend="tensorflow") # produces a tn.Tensor with the TensorFlow backend
`

**Comment by MikeMerz** [comment]
@alewis Hey, I tried running the snippet in the last comment and it seems to work fine, did someone already write this?
